### PR TITLE
0.0.2 release

### PR DESCRIPTION
**Pull request by eppv**
library 0.0.2 release to use as package.

### CODE REVIEW

**Review by AntonVeldin — DISMISSED** [review]

**Review by AntonVeldin — COMMENTED** [review]
Nice fw

**Review by eppv — COMMENTED** [review]
this is like one of the first steps

File: tests/test_database_extractors.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#     limitations under the License.
+
+import pytest
+from witness.extractors import ODBCExtractor
+
+xfail = pytest.mark.xfail
+parametrize = pytest.mark.parametrize
+
+# region Mock
+
```
Comment (eppv) [inline tests/test_database_extractors.py]: remove hardcoded mock data from test module

**Comment by eppv** [comment]
PR merged and closed due to the correction of all really important shortcomings.

**Pull request merged**